Source/WebKit/UIProcess/API/glib/WebKitURISchemeRequest.cpp

```diff
@@ -204,6 +205,26 @@ SoupMessageHeaders* webkit_uri_scheme_request_get_http_headers(WebKitURISchemeRe
     return request->priv->headers.get();
 }
 
+/**
+ * webkit_uri_scheme_request_get_http_body:
+ * @request: a #WebKitURISchemeRequest
+ *
+ * Get the request body.
+ *
+ * Returns: (transfer none): the body of the @request.
```

Comment:
Should either (a) change the implementation to ensure it does not return NULL, or (b) mark it as nullable:

```suggestion
 * Returns: (transfer none): (nullable): the body of the @request.
```

Reply:
formData could be empty, so I'll choose nullable.